### ISSUE TITLE
Ensure tests work in browser and node

### ISSUE THREAD

**Issue by ashleygwilliams** [rmurphey/js-assessment#97]
`npm test` does not currently work. The assessment tests should be able to be run both in the browser, and from the command line with the `npm test` command.

**Comment by kadamwhite** [comment]
Additionally, `path.existsSync` was deprecated and removed in Node 0.12 and IO. The testing harness should be upgraded to be able to be used in these versions.

**Comment by ashleygwilliams** [comment]
`npm test` throws this error in both 0.12.2 and iojs-1.7.1

```
Now using node v0.12.2
(jessie)ag_dubs@localhost:~/js-assessment$ npm test

> js-assessment@0.1.0 test /home/ag_dubs/js-assessment
> ./node_modules/mocha/bin/mocha -R spec tests/runner.js

child_process: customFds option is deprecated, use stdio instead.
/home/ag_dubs/js-assessment/node_modules/requirejs/bin/r.js:2267
        if (path.existsSync(url)) {
```

**Comment by ashleygwilliams** [comment]
gonna work on updating the require dependency and see how it goes. currently using `1.0.x`, latest release is `2.1.17`

**Comment by jacobroufa** [comment]
This should be fixed with #112. :+1:

**Comment by ashleygwilliams** [comment]
solved with #112, closing
